conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -55,7 +55,9 @@ install:
     # Tell conda we want an updated version of conda-build and {{ " ".join(remote_ci_setup) }}
     - cmd: conda.exe install -n root -c conda-forge --quiet --yes conda-build pip {{ " ".join(remote_ci_setup) }}
     {%- if local_ci_setup %}
-    - cmd: conda.exe uninstall --quiet --yes --force {{ " ".join(remote_ci_setup) }}
+    # Uninstall the conda-forge-ci-setup package but leave its dependencies
+    # so that we may bootstrap the installation process
+    - cmd: conda.exe uninstall --quiet --yes --force conda-forge-ci-setup
```

Comment:
This was an option so the package could be renamed. We should preserve that.

Reply:
The 3rd option is to leave it as is.

The current "state" works for tensorflow, and works for `conda-forge-ci-setup`. This is "good enough" for me.

Reply:
We actually have the version as part of the option: https://github.com/conda-forge/conda-smithy/blob/c299172b8e86f16b94d5f02234e89df35c5440db/conda_smithy/configure_feedstock.py#L1759

So smithy has been using a mix of things. 

I don't have strong feelings here, but it seems like if folks want to add extra ci setup packages, then we should have added a new optional originally called "extra_ci_setup" or similar. 

In any case, most things are half-broken most of the time. I'll defer to the rest of core here. 

cc @conda-forge/core

Reply:
I had originally proposed something like `extra`, it was rejected. I think we hadn't considered the bootstrap procedure. Selfishly speaking, i don't have many reasons to push this PR forward

Reply:
That is totally fine. Let's close this one and leave things as they are. We can go back and fix later by making the conda-forge specific stuff a separate option since it is special anyways. Thank you for all of the hard work here!

Reply:
xref: https://github.com/conda-forge/conda-smithy/issues/1597